pkg/host/machine_info.go

```diff
@@ -38,8 +38,16 @@ func CollectModulesInfo() ([]KernelModule, error) {
 	return machineModulesInfo()
 }
 
+func CollectDirsInfo(globs []string) (map[string][]string, error) {
```

Comment:
In the descriptions it's called "glob", in prog.Target it's called "files" and here it's "dirs". Please stick to consistent terminology throughout the change.